### PR TITLE
Summarize unassigned department's pins as "None" deparment

### PR DESCRIPTION
**Pull request by parnurzeal**

### CODE REVIEW

File: test/services/summarize-state/index.test.js [hunk 1]
```diff
@@ -89,6 +89,14 @@ describe('state summary service', () => {
             [pinStates.RESOLVED]: 0,
             [pinStates.REJECTED]: 0,
           },
+          None: {
+            [pinStates.VERIFIED]: 0,
+            [pinStates.UNVERIFIED]: 0,
+            [pinStates.ASSIGNED]: 0,
+            [pinStates.PROCESSING]: 1,
```
Comment (supasate) [inline test/services/summarize-state/index.test.js]: Pins that are not assigned cannot be in PROCESSSING state.

File: test/fixtures/pins.js [hunk 2]
```diff
@@ -52,4 +54,17 @@ module.exports = [
...
+    _id: ObjectId(PIN_PROCESSING_ID), // eslint-disable-line new-cap
+    created_time: '2016-12-03',
+    detail: PIN_PROCESSING_DETAIL,
+    organization: ORGANIZATION_ID, // organization ObjectId
+    owner: '579334c75563625d6281b6f1', // adminUser ObjectId
+    provider: '579334c75563625d6281b6f1', // adminUser ObjectId
+    location: {
+      coordinates: [100.56983534305, 13.730537951107],
+    },
+    status: 'processing',
```
Comment (supasate) [inline test/fixtures/pins.js:66]: Unassigned pin cannot be in processing state.

File: test/fixtures/pins.js [hunk 3]
```diff
@@ -52,4 +54,17 @@ module.exports = [
     status: 'assigned',
     is_archived: false,
   },
+  {
+    _id: ObjectId(PIN_PROCESSING_ID), // eslint-disable-line new-cap
```
Comment (supasate) [inline test/fixtures/pins.js:56]: Unassigned pin cannot be in processing state.

File: test/fixtures/pins.js [hunk 4]
```diff
@@ -52,4 +54,17 @@ module.exports = [
...
+    created_time: '2016-12-03',
+    detail: PIN_PROCESSING_DETAIL,
```
Comment (supasate) [inline test/fixtures/pins.js:59]: Unassigned pin cannot be in processing state.

File: test/fixtures/constants.js [hunk 5]
```diff
@@ -7,8 +7,8 @@ const ORGANIZATION_ID = '57933111556362511181aaa1';
 // Pins
 const PIN_ASSIGNED_ID = '579334c75563625d6281b6f8';
 const PIN_ASSIGNED_DETAIL = 'This is an assigned pin';
-const PIN_PROCESSING_ID = '579334c75563625d6281b6a5';
-const PIN_PROCESSING_DETAIL = 'This is an processing pin';
+const PIN_DEPARTMENT_UNASSIGNED_ID = '579334c75563625d6281b6a5';
+const PIN_DEPARTMENT_UNASSIGNED_DETAIL = 'This is an department-unassigned (unverified) pin';
```
Comment (supasate) [inline test/fixtures/constants.js]: Use existing `PIN_UNVERIFIED_ID/DETAIL`.

File: test/fixtures/pins.js [hunk 6]
```diff
@@ -55,16 +55,16 @@ module.exports = [
     is_archived: false,
   },
   {
-    _id: ObjectId(PIN_PROCESSING_ID), // eslint-disable-line new-cap
+    _id: ObjectId(PIN_DEPARTMENT_UNASSIGNED_ID), // eslint-disable-line new-cap
```
Comment (supasate) [inline test/fixtures/pins.js]: Use existing `PIN_UNVERIFIED_ID`.

File: test/fixtures/pins.js [hunk 7]
```diff
@@ -55,16 +55,16 @@ module.exports = [
...
     created_time: '2016-12-03',
-    detail: PIN_PROCESSING_DETAIL,
+    detail: PIN_DEPARTMENT_UNASSIGNED_DETAIL,
```
Comment (supasate) [inline test/fixtures/pins.js]: Use existing `PIN_UNVERIFIED_DETAIL`.

**Comment by parnurzeal** [comment]
As discussed, we cannot use the existing id because it will be duplicate.

I have corrected the pin mock data to have a correct assigned department, instead.

**Comment by supasate** [comment]
LGTM

**Pull request merged**